support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -180,6 +180,17 @@ object AddressAndCurrencyValidationRules {
     ) {
       stateFromRequest.isDefined.otherwise(s"state is required for $countryFromRequest")
     } else Valid
+  def hasValidPostcodeLength(postcodeFromRequest: Option[String], addressType: String): Result = {
+    val validPostCode = postcodeFromRequest match {
+      case Some(postCode) if (postCode.length <= 20) => true
+      case None => true
+      case _ => false
+    }
+    if (validPostCode)
+      Valid
+    else
+      Invalid(s"$addressType PostCode length must be less than 20 characters")
```

Comment:
I think this message should actually say “… must be 20 characters or fewer”, or “… must be less than 21 characters”, since 20 characters is an acceptable length.

Reply:
Changed the Invalid message to PostCode length must not be greater than 20 characters